core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalBackendConf.scala

```diff
@@ -90,8 +90,12 @@ trait ExternalBackendConf extends SharedBackendConf {
   }
 
   def setClusterSize(clusterSize: Int) = set(PROP_EXTERNAL_CLUSTER_SIZE._1, clusterSize.toString)
-
-  def setClientConnectionTimeout(timeout: Int) = set(PROP_EXTERNAL_CLIENT_CONNECTION_TIMEOUT._1, timeout.toString)
+  @DeprecatedMethod
+  def setClientConnectionTimeout(timeout: Int) = {
+    logWarning(s"Setting the ${PROP_EXTERNAL_CLIENT_CONNECTION_TIMEOUT._1} has not effect and is not required" +
```

Comment:
nit: Shouldn't the method name be mentioned in the warning message?

Reply:
👍